### PR TITLE
Feature/sahu/documentation update

### PR DESCRIPTION
**Pull request by danielmohansahu**
Small documentation clarifications / corrections.

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -64,21 +64,15 @@ To generate CPPCheck and CPPLint code analysis:
 
 ```bash
 # from the BashScripts directory
```
Comment (selyard) [inline readme.md]: These are all currently written to be run from the BashScripts directory; running from the root directory will cause the build to crash. Can rewrite to call from outside of directory, but figured this was just as easy.

**Review by selyard — CHANGES_REQUESTED** [review]
From "Midterm Phase 1 Overall Project Guidelines":

- Begin with UML diagram designs (Class structure as well as activity/sequence). Add a “UML” directory to the repository and an “initial” subdirectory. Add UML files to the initial directory (PDF). If design iterations occur during implementation, revise UML diagrams and add to a “revised” sub-directory.

Need to keep originals in the folder with updated ones.

Also, currently bash scripts need to be run from within that folder; otherwise, it will look in the wrong spots for files.

File: readme.md (hunk 1 above)
Comment (danielmohansahu) [reply readme.md]: That's my bad; I totally missed the comment in the code. I'll revert my changes.

**Comment by danielmohansahu** [comment]
> From "Midterm Phase 1 Overall Project Guidelines":
> 
>     * Begin with UML diagram designs (Class structure as well as activity/sequence). Add a “UML” directory to the repository and an “initial” subdirectory. Add UML files to the initial directory (PDF). If design iterations occur during implementation, revise UML diagrams and add to a “revised” sub-directory.
> 
> 
> Need to keep originals in the folder with updated ones.
> 
> Also, currently bash scripts need to be run from within that folder; otherwise, it will look in the wrong spots for files.

Got it; I'll need to change the folder names to get the images to show up properly in the readme, but other than that I'll leave it as before.

**Review by selyard — APPROVED** [review]

**Pull request merged**